app/assets/stylesheets/custom.css.scss

```diff
@@ -16,9 +16,12 @@ body {
   font-family: Avenir;
   color: $site-black;
   background-color: $site-white;
-  padding-top: 70px;
 }
 
+.container { padding-top: 46px; }
+
+.container.nav { padding-top: 0; }
```

Comment:
Don't know how to do this. This fucks it up:

```css
.container { padding-top: 46px;
    .nav { padding-top: 0; }
}
```
Hound is bitch about using id selectors, and I can't figure out another way around it. If any of you can then I'm all ears. I'm really getting sick of CSS's shit.